src/classes/metadata/pdf.vala

```diff
@@ -444,6 +444,8 @@ namespace pdfpc.Metadata {
                             note_text += a.get_contents();
                             break;
                     }
+                    // Remove the annotation to avoid its rendering
+                    page.remove_annot(a);
```

Comment:
This call needs at least poppler >= 0.22. We should have a clean version dependency on that.